docs/examples/use_cases/webdataset-externalsource.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<p>in the Data Representation, the drive access is improved twice (a bit of repetition). </p><p> </p><p>Maybe:</p><p>This approach improves drive access latency a lot, since the files are kept on the disk in contiguous container rather than separate, fragmented files. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>